source/index.txt

```diff
@@ -10,6 +12,96 @@ MongoDB Shell
    :depth: 1
    :class: singlecol
 
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+The |mdb-shell| is an interactive JavaScript shell interface to MongoDB,
+which provides a powerful interface for users to test queries and
+operations directly with the database. The |mdb-shell| also provides a
+fully functional JavaScript environment for use with MongoDB.
+
+The |mdb-shell| is available as a standalone package in the MongoDB
+download center.
+
+Download and Install the |mdb-shell|
+------------------------------------
+
+.. tabs-platforms::
+
+   .. tab::
+      :tabid: windows
+
+      Hello, I am Windows content.
+
+   .. tab::
+      :tabid: macOS
+
+      Hello, I am macOS content.
+
+   .. tab::
+      :tabid: linux
+      
+      Hello, I am Linux (rpm / deb) content.
+
+Multi-Line Operations in the |mdb-shell|
+----------------------------------------
+
+Run the ``editor()`` command inside of the |mdb-shell| to open the
+editor and manage comprehensive multiline functions:
+
+.. example::
+
+   Run the following command within the |mdb-shell|:
+
+   .. code-block:: javascript
+
+      editor()
+
+   The |mdb-shell| enters editor mode:
+
+   .. code-block:: javascript
+
+      // Entering editor mode (^D to finish, ^C to cancel)
+      const { Writable } = require('stream');
+
+        const myWritable = new Writable({
+          write(chunk, encoding, callback) {
+            // ...
+          },
+          writev(chunks, callback) {
+            // ...
+          }
+        });
+
+.. note::
+
+   Currently, ``editor()`` is the only way to run multiline commands in
```

Comment:
```suggestion
   Currently, ``editor()`` is the only way to paste multiline commands in
```

Reply:
You can do multiline editing also without `.editor`, it's pasting a multiline command/script that does not work.